src/components/GameOverShroud.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="game-over-shroud">
+
+    <div>
+      Game Over
+    </div>
+
+    <div>
+      Score: {{score}}
+    </div>
+
+    <div v-on:click="newGame" class="new-game-shroud-button">
+      Try Again
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
```

Comment:
how do you export from an inline script tag? This is weird because I dont think you can import this from anywhere so there shouldnt be a need to export